### PR TITLE
Update communicate-with-users-from-other-organizations.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/communicate-with-users-from-other-organizations.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ appliesto:
 localization_priority: Priority
 description: Learn how to call, chat, find, and add users from outside the organization in Microsoft Teams using external access (federation) and guest access.
 ---
-
+>! Note for teams free licenses, external couminication is not supported.
```
Comment (JohanFreelancer9) [inline Teams/communicate-with-users-from-other-organizations.md:22]:
```suggestion
```

File: Teams/communicate-with-users-from-other-organizations.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ appliesto:
...
 # Call, chat, and collaborate with people outside your organization in Microsoft Teams
```
Comment (JohanFreelancer9) [inline Teams/communicate-with-users-from-other-organizations.md:24]: > [!NOTE]
> Microsoft Teams free licenses do not support external communication.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@scanum this PR has unresolved edits.

@RuiPereiraTabares what's the scope of "external communications"? I'm not clear on what feature you're referring to.

**Comment by RuiPereiraTabares** [comment]
@MikePlumleyMSFT  TeamsFree is not going to provision a cloudsipproxxyadress or either the domain will be online sip enabled, therefore the externalcomunications will not work for it. 
Teamsfree is different than the exploratory licenses is the completly freemiun subscription 
https://support.microsoft.com/en-us/office/differences-between-microsoft-teams-and-microsoft-teams-free-0b69cf39-eb52-49af-b255-60d46fdf8a9c
we do not mention this, However over my tests in deepsite i could see that is not provisioned, what will make imposible to comunicate externally or make a external search to a TeamsFree license

**Pull request merged**